packages/hop-node/src/watchers/BondTransferRootWatcher.ts

```diff
@@ -63,6 +63,15 @@ class BondTransferRootWatcher extends BaseWatcher {
         transferIds
       } = dbTransferRoot
 
+      const availableCredit = this.getAvailableCreditForBond(destinationChainId!) // eslint-disable-line @typescript-eslint/no-non-null-assertion
+      if (availableCredit.lt(totalAmount!)) { // eslint-disable-line @typescript-eslint/no-non-null-assertion
+        this.logger.debug(
```

Comment:
I think it makes sense to define the `logger` here with the `transferRootId`, similar to the [BondWithdrawalWatcher](https://github.com/hop-protocol/hop/blob/33384d0da7f1e6de16ff80fc6cbc018067c91de3/packages/hop-node/src/watchers/BondWithdrawalWatcher.ts#L74) for easier debugging.

Reply:
Resolved with https://github.com/hop-protocol/hop/pull/328/commits/5d98c91084f4ba98b4c6cb907f3a6269a380be8f